packages/cli/generation/ir-generator/src/utils/getAvailability.ts

```diff
@@ -0,0 +1,28 @@
+import { Availability, AvailabilityStatus } from "@fern-api/ir-sdk";
+
+type RawAvailabilityStatus = "in-development" | "pre-release" | "generally-available" | "deprecated";
+type RawAvailability = {
+    status: RawAvailabilityStatus;
+    message?: string;
+};
+
+export function getAvailability(
```

Comment:
I think you might be able to re-use the functions here no? https://github.com/fern-api/fern/blob/main/packages/cli/generation/ir-generator/src/converters/convertDeclaration.ts#L40